### PR TITLE
VoxelSniper: The journey through the Fabric of time. Part one: Upgrading ForgeGradle

### PR DESCRIPTION
**Pull request by KevinDaGame**
## What type of PR is this? (check all applicable)

- [x] Feature
- [ ] Refactor
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Upgrade Forgegradle to 6
Upgrade gradle from 7.x to 8.1.1

https://docs.gradle.org/current/userguide/upgrading_version_7.html
https://gist.github.com/ChampionAsh5357/98b08c117cb2057d8029a8919c378a4c

## QA Instructions, Screenshots, Recordings
It just works

## Added/updated tests?

- [ ] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
  have not been included_
- [ ] I need help with writing tests
- [x] Not needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: VoxelSniperForge/build.gradle [hunk 1]
```diff
@@ -1,5 +1,5 @@
 plugins {
-    id 'net.minecraftforge.gradle' version '5.1.+'
+    id 'net.minecraftforge.gradle' version '[6.0,6.2)'
```
Comment (Lennart99) [inline VoxelSniperForge/build.gradle:2]: Shouldn't this be one fixed version instead of a range? (I assume that is what this means..)

**Review by Lennart99 — APPROVED** [review]
LGTM

**Comment by Lennart99** [comment]
nice title btw, I like it ;)

File: VoxelSniperForge/build.gradle (hunk 1 above)
Comment (KevinDaGame) [reply VoxelSniperForge/build.gradle:2]: This is straight from the upgrade guide. It should be correct

Comment (KevinDaGame) [reply VoxelSniperForge/build.gradle:2]: For further questions, I refer you to the gif in the description

Comment (Lennart99) [reply VoxelSniperForge/build.gradle:2]: ok

**Pull request merged**